res/xml/qwertz.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<keyboard>
+  <row>
+    <key key0="q" key1="esc" key2="1" key3="\@" key4="!"/>
+    <key key0="w" key1="^" key2="2" key3="&quot;" key4="°"/>
+    <key key0="e" key1="§" key2="3" key4="€"/>
```

Comment:
`€` is already `Fn+$`. But perhaps we can keep it here anyway.

Reply:
Yes, please! It's needed frequently.